packages/cache/src/LocalCacheStorage.ts

```diff
@@ -14,45 +13,28 @@ export class LocalCacheStorage extends CacheStorage {
     return path.join(this.internalCacheFolder, hash);
   }
 
-  protected async _fetch(
-    hash: string,
-    outputFolder: string | string[]
-  ): Promise<boolean> {
+  protected async _fetch(hash: string): Promise<boolean> {
     const localCacheFolder = this.getLocalCacheFolder(hash);
 
     if (!fs.pathExistsSync(localCacheFolder)) {
       return false;
     }
 
-    await Promise.all(
-      outputFolderAsArray(outputFolder).map(async folder => {
-        const locationInCache = path.join(localCacheFolder, folder);
-
-        if (!fs.pathExistsSync(locationInCache)) {
-          return;
-        }
-
-        await fs.mkdirp(folder);
-        await fs.copy(locationInCache, folder);
-      })
-    );
+    fs.readdirSync(localCacheFolder).forEach(fileOrFolder => {
+      fs.copySync(path.join(localCacheFolder, fileOrFolder), fileOrFolder);
```

Comment:
Will `copySync` merge the content of, say, a cached folder and the destination folder? What if the glob was something like `["src/**/*.less"]`. Now we would have a `src` folder in our cache. When it copies this folder to the destination, will it merge or replace?

Reply:
I will replace, not sure what you mean by 'merge'?

Reply:
Yea, a bit hard to explain. Let me try through an example. `foo` has the following files 

foo/package.json
foo/src/index.js
foo/src/style/bar.less
foo/lib/...

We backfill ["lib", "src/**/*.less"]. The cache folder will now contain:

cache/foo/src/style/bar.less
cache/foo/lib/...

When we run `fs.readdirSync(localCacheFolder)`, I assume it will return `lib` and `src`. When we then run `fs.copySync(path.join("cache/foo", "src"), "foo");` I worry that we will replace the original `src` with the small subset of `src` that is in the cache. In this example, we would loose `foo/src/index.js`.

Maybe it would be better to use fast-glob and just enumerate all files in the cache and copy each one?

Reply:
I think this is how it is in the latest iteration, can you double check?